components/line/sources/new-message-received/new-message-received.mjs

```diff
@@ -0,0 +1,69 @@
+import line from "../../line.app.mjs";
+import { axios } from "@pipedream/platform";
+
+export default {
+  key: "line-new-message-received",
+  name: "New Message Received",
+  description: "Emit new event for every received message in a channel. [See docs here](https://developers.line.biz/en/docs/messaging-api/building-bot/#page-title)",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    line,
+    db: "$.service.db",
+    http: "$.interface.http",
+    channelAccessToken: {
+      propDefinition: [
+        line,
+        "channelAccessToken",
+      ],
+    },
+  },
+  hooks: {
+    async activate() {
+      try {
+        await this.checkWebhook();
```

Comment:
What's the intent of checking the webhook here if the return value is not used? Is this GET needed on the API's side before calling the next method?

Reply:
The reason is that if the user hasn't yet enabled webhooks in the Developer Console, the request will throw an error. Then it can log correct help message

Reply:
Gotcha, makes sense!